### PR TITLE
Makefile: IMAGES=foo,bar for origin-release target to replace multiple images

### PR DESCRIPTION
**Pull request by sttts**
Example to replace the control-plane completely:

```
IMAGES=cluster-kube-apiserver-operator,cluster-kube-controller-manager-operator,cluster-kube-scheduler-operator DOCKER_ORG=sttts make origin-release
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 GOFLAGS :=
-DOCKER_ORG ?= $(USER)
+DOCKER_ORG ?= openshift
```
Comment (soltysh) [inline Makefile]: Same as in the other one, please no docker org.

File: Makefile [hunk 2]
```diff
@@ -27,21 +27,24 @@ endif
...
 
 clean:
 	$(RM) ./cluster-kube-apiserver-operator
 .PHONY: clean
 
+, := ,
+IMAGES ?= cluster-kube-apiserver-operator
+QUOTED_IMAGES=\"$(subst $(,),\"$(,)\",$(IMAGES))\"
+
 origin-release:
```
Comment (tnozicka) [inline Makefile]: not excited about this chain, but I get why - let me see if I can come up with something cleaner

Comment (tnozicka) [reply Makefile]: also given part of it is preexisting you don't actually need to wait

**Comment by mfojtik** [comment]
```
[mfojtik@dev-centos cluster-kube-apiserver-operator]$ make origin-release
docker pull registry.svc.ci.openshift.org/openshift/origin-release:v4.0
Trying to pull repository registry.svc.ci.openshift.org/openshift/origin-release ...
v4.0: Pulling from registry.svc.ci.openshift.org/openshift/origin-release
aeb7866da422: Already exists
def284b63635: Pull complete
3a8744891aa5: Pull complete
1b8329a83630: Pull complete
Digest: sha256:6b9b28b933f27926afb26c12af64a1762612a1b31897d878748822490f0fb51e
Status: Downloaded newer image for registry.svc.ci.openshift.org/openshift/origin-release:v4.0
bash -c 'docker build -f <(sed "s/DOCKER_ORG/openshift/g;s/IMAGES/\"cluster-kube-apiserver-operator\"/g" hack/lib/Dockerfile-origin-release) -t "openshift/origin-release:latest" hack'
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /proc/8724/fd/pipe:[52303]: no such file or directory
make: *** [origin-release] Error 1
```

File: Makefile [hunk 3]
```diff
@@ -27,21 +27,24 @@ endif
...
 	docker pull registry.svc.ci.openshift.org/openshift/origin-release:v4.0
-	bash -c 'docker build -f <(sed "s/DOCKER_ORG/$(DOCKER_ORG)/" Dockerfile-origin-release) -t "$(DOCKER_ORG)/origin-release:latest" .'
+	bash -c 'docker build -f <(sed "s/DOCKER_ORG/$(DOCKER_ORG)/g;s/IMAGES/$(QUOTED_IMAGES)/g" hack/lib/Dockerfile-origin-release) -t "$(DOCKER_ORG)/origin-release:latest" hack'
```
Comment (mfojtik) [inline Makefile]: can you make this `imagebuilder build -f` ?

File: Makefile [hunk 4]
```diff
@@ -27,21 +27,24 @@ endif
...
 	docker push $(DOCKER_ORG)/origin-release:latest
 	@echo
 	@echo "To install:"
 	@echo
-	@echo "  make images"
-	@echo "  docker tag openshift/origin-cluster-kube-apiserver-operator $(DOCKER_ORG)/origin-cluster-kube-apiserver-operator"
+	@echo "  DOCKER_ORG=$(DOCKER_ORG) make images"
```
Comment (mfojtik) [inline Makefile]: don't you need this before running this so you have the correct SHA?

File: hack/lib/Dockerfile-origin-release [hunk 5]
```diff
@@ -3,7 +3,7 @@ FROM registry.svc.ci.openshift.org/openshift/origin-release:v4.0 as origin-relea
 FROM fedora:29 as jq
 RUN yum install -y jq
```
Comment (mfojtik) [inline hack/lib/Dockerfile-origin-release]: this is pretty slow to me on rebuilds, I guess we want some custom `origin-build-tools` image that already have the  `jq` installed.

File: Makefile (hunk 4 above)
Comment (sttts) [reply Makefile]: no, we use latest, no SHA

File: Makefile (hunk 3 above)
Comment (sttts) [reply Makefile]: done

File: Makefile (hunk 1 above)
Comment (sttts) [reply Makefile]: done

**Comment by sttts** [comment]
@mfojtik @tnozicka @soltysh addressed all comments (with the jq exception, @mfojtik is working on that)

File: Makefile [hunk 6]
```diff
@@ -1,5 +1,5 @@
 GOFLAGS :=
-DOCKER_ORG ?= $(USER)
+IMAGE_REPOSITORY_NAME ?= openshift
```
Comment (tnozicka) [inline Makefile]: as we have started the bikeshedding I don't think this is the correct nomenclature

repository is openshift/release, this is an org

**Comment by mfojtik** [comment]
I would move this into openshift/master-operator-sdk ?

**Comment by sttts** [comment]
@mfojtik we agreed to have this here first. Follow-up to move it whereever we want. Lgty?

**Pull request merged**